src/main/java/com/minecolonies/blockout/controls/Scrollbar.java

```diff
@@ -56,6 +62,13 @@ public Scrollbar(final ScrollingContainer container, final PaneParams params)
     {
         this(container);
         //  TODO: Parse Scrollbar-specific Params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)